src/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from "react";
+import classnames from "classnames";
+
+import "./_aside.scss";
+
+type Props = {
+  children: ReactElement | ReactElement[];
+  width?: "wide" | "narrow";
+  pinned?: boolean;
+};
+
+export default function Aside({
+  children,
+  width = undefined,
+  pinned = false,
+}: Props): ReactElement {
```

Comment:
Sorry we've standardized on `JSX.Element` after some research and discussion (also the other projects use it as well) If you wouldn't mind updating these files with that 👍